core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -69,17 +76,28 @@ oppia.controller('Dashboard', [
       $scope.checkForMobileView();
     });
 
+    $scope.sortExplorationsList = function(sortType) {
```

Comment:
The behavior of this is a bit weird -- every time I change the sort function, it changes the ascending/descending-ness of the sort. `sortType` is also being used as an optional parameter, which is not ideal. I suggest using 2 functions, one to change the sort key and one to change the sort order.